app/decorators/manageiq/providers/storage_manager_decorator.rb

```diff
@@ -29,8 +29,8 @@ def quadicon
         :tooltip => n_("%{number} Cloud Volume", "%{number} Cloud Volumes", t) % {:number => t}
       }
       icon[:top_right] = {
-        :text    => t = cloud_volume_snapshots.size,
-        :tooltip => n_("%{number} Cloud Volume Snapshot", "%{number} Cloud Volume Snapshots", t) % {:number => t}
+        :text    => t = storage_resources.size,
+        :tooltip => n_("%{number} Storage Pools", "%{number} Storage Pools", t) % {:number => t}
```

Comment:
I think the first parameter here needs to be the singular (so `Storage Pool`) @skateman Please correct me if I'm wrong

Reply:
fixed